.azure/templates/steps/system_test_general.yaml

```diff
@@ -145,7 +145,7 @@ jobs:
 
       env:
         DOCKER_TAG: $(docker_tag)
-        BRIDGE_IMAGE: "latest-released"
+        BRIDGE_IMAGE: "quay.io/ppatierno/kafka-bridge:bridge-config"
```

Comment:
We need to address this before merging this.

Reply:
Yes sure.